### DISCUSSION TITLE
comparisons/weaviate-vs-chroma

### DISCUSSION THREAD

**Discussion by giscus** [zackproser/portfolio#566] [Announcements]
# comparisons/weaviate-vs-chroma

A detailed comparison of the Weaviate and Chroma vector databases

https://zackproser.com/comparisons/weaviate-vs-chroma

**Comment by H4125H** [comment]
Best VDB comparison table available: https://superlinked.com/vector-db-comparison

Powered by, https://superlinked.com/
